src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ExecutableDebugPropertyPage.xaml

```diff
@@ -77,4 +77,7 @@
                 DisplayName="Enable SQL Server debugging"
                 Description="Enable debugging of SQL scripts and stored procedures." />
 
+  <BoolProperty Name="WebView2Debugging"
+                DisplayName="Enable WebView2 debugging"
+                Description="Enable JavaScript debugger for Microsoft Edge (Chromium) based WebView2. Requires the JavaScript Diagnostics component." />
```

Comment:
I wonder if we could have a visibility condition based upon VS workload/component availability...

Reply:
Yeah, there are definitely improvements we could make here, but this brings us more or less back up to parity with the experience in the old property pages.

Reply:
Sure, it was more of a forward thinking idea. It seems like it would be a useful facility.

Reply:
Tracked by https://github.com/dotnet/project-system/issues/7358